u2f-gae-demo/war/js/u2f-api.js

```diff
@@ -709,7 +717,7 @@ u2f.responseHandler_ = function(message) {
  * @param {number=} opt_timeoutSeconds
  */
 u2f.sign = function(appId, challenge, registeredKeys, callback, opt_timeoutSeconds) {
-  if (js_api_version === undefined) {
+  if (js_api_version === undefined && !u2f.isIOSChrome_()) {
```

Comment:
To avoid jumping back and forth between browser and iOS client app, it's better to decide the JS version without going to the app. For this pull request, I only moved the iOS support code to the correct file, while it is still using JS 1.0. After this pull request, I will upgrade the JS support in both the JS and iOS client app to 1.1